service/src/main/java/org/folio/rest/workflow/service/WorkflowEngineService.java

```diff
@@ -47,11 +47,8 @@ public class WorkflowEngineService {
   @Value("${okapi.url}")
   private String okapiUrl;
 
-  @Value("${okapi.camunda.base-path}")
-  private String basePath;
-
-  @Value("${okapi.camunda.rest-path}")
-  private String restPath;
+  @Value("${okapi.workflow-engine.path:/camunda}")
```

Comment:
This must match mod-camunda property `spring.jersey.application-path`.

https://github.com/folio-org/mod-camunda/pull/122/files#diff-b4c0421bcb3280a8e738550c830dba197057c515bcdd9579a52cbfe9adca2061R108